plugins/smart_proxy_reports/control

```diff
@@ -0,0 +1,23 @@
+Source: ruby-smart-proxy-reports
+Section: ruby
+Priority: optional
+Maintainer: Lukas Zapletal <lzap+pub@redhat.com>
+Build-Depends: debhelper (>= 11~),
+               ruby-rack (>= 1.1.0),
```

Comment:
this is clearly wrong:

https://github.com/theforeman/smart_proxy_reports/blob/ce29f1b57b0090945e8c999debf9a58cd7390331/smart_proxy_reports.gemspec#L18-L19